connectors/class-connector-jetpack.php

```diff
@@ -372,16 +372,17 @@ public function callback_jetpack_log_entry( array $entry ) {
 				$user_id = get_current_user_id();
 			}
 
-			$user       = new \WP_User( $user_id );
-			$user_email = $user->user_email;
-			$user_login = $user->user_login;
-			$context    = 'users';
-			$action     = $method;
-			$meta       = compact( 'user_id', 'user_email', 'user_login' );
-			$message    = sprintf(
-				/* translators: %1$s: a user display name, %2$s: a status and the connection either "from" or "to" (e.g. "Jane Doe", "unlinked from") */
+			$user              = new \WP_User( $user_id );
+			$user_email        = ! empty( $user->user_email ) ? $user->user_email : '';
+			$user_login        = ! empty( $user->user_login ) ? $user->user_login : '';
+			$user_display_name = ! empty( $user->display_name ) ? $user->display_name : '';
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question**
Could a user without a display name be something involving the `$user_id` here? Like 'Unknown user, user_id: 21'

Reply:
@delawski If there's no user, like the user has been deleted, then it'd be nice to have the display name be shown as something like "Unknown user {$user_id}"? I think we did this somewhere else too, it's the same sort of thing.

Reply:
Now I get it, thank you for explaining :) I've addressed it in 080e85fc.